templates/k8s/cakeshop.yaml.erb

```diff
@@ -64,11 +72,32 @@ spec:
         - name: cakeshop
           image: <%= @Cakeshop_Docker_Repo %>/cakeshop:<%= @Cakeshop_Version %>
           imagePullPolicy: Always
-          env:
-            - name: JAVA_OPTS
-              value: "-Dgeth.auto.start=false -Dgeth.auto.stop=false"
+          command: [ "sh" ]
+          args:
+            - "-cx"
+            - "
+               cp  /etc/cakeshop/cakeshop-nodes.json /cakeshop/cakeshop-nodes.json;
+              <%- @nodes.each do |node| -%>
+               <%= set_node_template_vars(node) -%>
+  sed -i \"s/<%= @Service_Prefix %>_SERVICE_HOST/$<%= @Service_Prefix %>_SERVICE_HOST/g\" /cakeshop/cakeshop-nodes.json;
```

Comment:
Minor tweak: 
Moving `<%= set_node_template_vars(node) -%>` next to `<%- @nodes.each do |node| -%>` will make the output in `out/07-cakeshop.yaml` line up.
```
              <%- @nodes.each do |node| -%><%= set_node_template_vars(node) -%>
               sed -i \"s/<%= @Service_Prefix -%>_SERVICE_HOST/$<%= @Service_Prefix -%>_SERVICE_HOST/g\" /cakeshop/cakeshop-nodes.json;
              <%- end -%>
```